docs/source/serialization.mdx

```diff
@@ -10,27 +10,38 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Exporting transformers models
+# Exporting 🤗 Transformers Models
 
-## ONNX / ONNXRuntime
+If you need to deploy 🤗 Transformers models in production environments, we
+recommend exporting them to a serialized format that can be loaded and executed
+on specialized runtimes and hardware. In this guide, we'll show you how to
+export 🤗 Transformers models in two widely used formats: ONNX and TorchScript.
 
-Projects [ONNX (Open Neural Network eXchange)](http://onnx.ai) and [ONNXRuntime (ORT)](https://microsoft.github.io/onnxruntime/) are part of an effort from leading industries in the AI field to provide a
-unified and community-driven format to store and, by extension, efficiently execute neural network leveraging a variety
-of hardware and dedicated optimizations.
+Once exported, a model can optimized for inference via techniques such as
+quantization and pruning. If you are interested in optimizing your models to run
```

Comment:
I point people to `optimum` here to indicate that `transformers` is only concerned with _exporting_ models, not optimising them.